lib/features/email/data/network/email_api.dart

```diff
@@ -266,15 +258,22 @@ class EmailAPI with HandleSetErrorMixin {
       .build()
       .execute();
 
-    final getEmailResponse = response.parse<GetEmailResponse>(
-      getEmailInvocation.methodCallId,
-      GetEmailResponse.deserialize);
+    final setEmailResponse = response.parse<SetEmailResponse>(
+      setEmailInvocation.methodCallId,
+      SetEmailResponse.deserialize,
+    );
+
+    final emailIdUpdated = setEmailResponse?.updated
+      ?.keys
+      .map((id) => EmailId(id))
+      .toList() ?? [];
+    final mapErrors = handleSetResponse([setEmailResponse]);
 
-    return Future.sync(() async {
-      return getEmailResponse!.list;
-    }).catchError((error) {
-      throw error;
-    });
+    if (emailIdUpdated.isNotEmpty) {
```

Comment:
We don't need to care about it. Because there are still some cases that are successful when `emailIdUpdated.isNotEmpty`. So in the interactor we will check if the number of emails passed and the number of emails updated match. There will be corresponding return values ​​for each case.